projects/github-actions/repo-gardening/src/tasks/triage-new-issues/index.js

```diff
@@ -19,21 +19,41 @@ async function hasPriorityLabels( octokit, owner, repo, number ) {
 }
 
 /**
- * Find specific plugin impacted by issue, based off issue contents.
+ * Find list of plugins impacted by issue, based off issue contents.
  *
  * @param {string} body - The issue content.
- * @returns {string} Plugin concerned by issue.
+ * @returns {Array} Plugins concerned by issue.
  */
-function findPlugin( body ) {
-	const regex = /###\sImpacted\splugin\n\n(\w*)\n/gm;
+function findPlugins( body ) {
+	const regex = /###\sImpacted\splugin\n\n([a-zA-Z ,]*)\n\n/gm;
 
 	let match;
 	while ( ( match = regex.exec( body ) ) ) {
-		const [ , plugin ] = match;
-		return plugin;
+		const [ , plugins ] = match;
+		return plugins.split( ', ' );
```

Comment:
Might want to filter out empty-string and space-only strings, to avoid potential confusion for the caller.
```suggestion
		return plugins.split( ', ' ).filter( v => v.trim() !== '' );
```